CONTRIBUTING.md

```diff
@@ -91,17 +91,18 @@ Fabric is an open source project 🦄 and as such depends on the **genuine effor
 
 - Read this section through.
 - Take a look at [**GOTCHAS**][gotchas]
-- Follow [Developing](#-developing-) and read [Testing](#-testing).
+- Follow [Developing](#-developing-) and [Testing](#-testing).
 
 ### ✅ Guidelines
 
+- **Be patient** \
+  Sometimes it takes time to get back to you. Someone eventually will. Having a small, concise and super clear change will make maintainers more prone to handle it quickly.
 - **Code Style** \
   Fabric uses [`prettier`][prettier] to format files and [`eslint`][eslint] for linting (`npm run lint -- --fix`).\
   To enjoy a seamless dev experience add the [`Prettier - Code formatter`][prettier_extension] extension via the extensions toolbar in VSCode.
-- **⛔ `dist`** \
-  Commit changes to [source files](src). Don't commit the generated [distribution files](dist).
+  Do not reorder imports. Irrelevant changes in a PR that are not created by prettier aren't needed nor welcome.
 - **Tests** \
-  PRs must be backed with relevant tests, follow [TESTING](#-testing).
+  PRs must be backed with relevant tests, follow [TESTING](#-testing). We are here to help.
```

Comment:
There are developer that do not do tests. I want to be clear, if you never wrote a test, ask for help. We are here to help is so generic.
Someone may not know the scope of a unit test, a rendering test or a more convolute integration test

Reply:
OK